### ISSUE TITLE
RCS1060 is not raised when declaring types in a file-scoped namespace

### ISSUE THREAD

**Issue by Arthri** [JosefPihrt/Roslynator#1475]
**Product and Version Used**: Roslynator.Analyzers version 4.7.0

**Steps to Reproduce**:
Have the following declarations and statements
```cs
namespace Z;

public class X {}

public class Y {}
```

**Actual Behavior**:
RCS1060 "declare each type in a separate file" is not raised

**Expected Behavior**:
RCS1060 is raised.

Furthermore, declaring them in the global namespace does raise RCS1060,
```cs
public class X {}

public class Y {}
```
and declaring them in a block-scope namespace also raises RCS1060
```cs
namespace Z
{
    public class X {}

    public class Y {}
}
```
